CHANGES.rst

```diff
@@ -242,7 +275,7 @@ Unreleased
   an instance of :class:`.Reddit`, and via the ``PRAW_REFRESH_TOKEN`` environment
   variable. To be prepared for PRAW 8, use the new :class:`.Reddit` keyword argument
   ``token_manager``. See :ref:`refresh_token` in PRAW's documentation for an example.
-- :meth:`.me` will no longer return ``None`` when called in :attr:`.read_only` mode
+- ``user.me`` will no longer return ``None`` when called in :attr:`.read_only` mode
```

Comment:
If we aren't going to link to it anymore should we show it as `reddit.user.me()` or ``Reddit.user.me()`?

Reply:
Good idea.